pkg/config/store.go

```diff
@@ -96,6 +113,8 @@ const (
 
 	// KMS
 	kmsSignerKMSRef = "signers.kms.kmsref"
+	// Fulcio
+	x509SignerFulcioEnabled = "signers.x509.fulcio.enabled"
```

Comment:
Should we have a way to specify the URL for fulcio?

Reply:
Also maybe a way to specify how to get auth (right now google, but going forward SPIFFE?)